client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+
+const MediaSettings = ( props ) => {
+	return (
+		<div>
+			<SectionHeader label={ translate( 'Media' ) }>
+				<Button
+					compact
+					primary
+					onClick={ props.submitFormAndActivateCustomContentModule }
+					disabled={ props.fetchingSettings || props.submittingForm }>
+					{ props.submittingForm ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+				</Button>
+			</SectionHeader>
+			<Card className="media-settings__card site-settings">
+				<FormFieldset>
+					<span style={ { 'float': 'right' } }>
```

Comment:
We should avoid such inline styles, let's write them in designated .scss files. And in general, the component will need some CSS love and a designer eye.